dataservice/util/data_gen/data_generator.py

```diff
@@ -0,0 +1,183 @@
+from datetime import datetime
+from dateutil import relativedelta
+import uuid
+import random
+
+from dataservice.util.data_gen.data import *
+from dataservice import create_app
+from dataservice.extensions import db
+from dataservice.api.participant.models import Participant
+from dataservice.api.participant.models import Participant
+from dataservice.api.demographic.models import Demographic
+from dataservice.api.diagnosis.models import Diagnosis
+from dataservice.api.sample.models import Sample
+from dataservice.api.aliquot.models import Aliquot
+from dataservice.api.sequencing_experiment.models import SequencingExperiment
+from dataservice.api.genomic_file.models import GenomicFile
+
+
+class DataGenerator(object):
```

Comment:
Since you have a class to represent the generator's state, why no use class attributes instead of global variables?